README.md

```diff
@@ -30,8 +30,8 @@
 
 # 📖 [webpack template] <a name="about-project"></a>
```

Comment:
- [ ] Lastly, kindly ensure that your README contains information specific to this project 😉 
<img width="767" alt="Screen Shot 2023-05-23 at 5 04 32 PM" src="https://github.com/gisachris/LeaderBoard/assets/102699958/0fc2877f-e36a-401a-9584-bf998662f329">